src/adversarial/neurify.jl

```diff
@@ -195,33 +195,23 @@ function get_max_nodewise_influence(nnet::Network,
     return (i_max, j_max, influence_max)
 end
 
-
-
-function forward_network(solver::Neurify, network::Network, input)
-    forward_network(solver, network, init_symbolic_grad(input))
-end
-function forward_network(solver::Neurify, network::Network, input::SymbolicIntervalGradient)
-    reachable = [input = forward_layer(solver, L, input) for L in network.layers]
-    return reachable
-end
-
-
-function forward_layer(solver::Neurify, layer::Layer, input)
-    return forward_act(solver, forward_linear(solver, input, layer), layer)
+function forward_network(solver::Neurify, network::Network, input::HPolytope)
+    println("in Neurify forward network")
+    return forward_network(solver, network, init_symbolic_grad(input))
```

Comment:
If we're going to redo forward_network as we are, let's get rid of definitions like these. We can do init_symbolic_grad directly in solve like we do for the first domain, and eliminate the need for this.

There is also a leftover debug statement 😉

Reply:
oh so we wouldn't even need this function? Just anywhere we call forward network we have to call init_symbolic_grad first?